src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -13,7 +11,7 @@ namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
     /// <summary>
     /// Abstract class used to buffer content returned by <see cref="ITagHelper"/>s.
     /// </summary>
-    public abstract class TagHelperContent : IEnumerable<string>, IHtmlContent
+    public abstract class TagHelperContent : IHtmlContent
```

Comment:
The test code for the MVC tag helpers is more convoluted given this change. This may be an indication we've made the `TagHelperContent` API harder to use. But :ok: until and unless some external tag helper author submits an issue.